plugin/src/main/java/net/dzikoysk/funnyguilds/rank/RankPlaceholdersService.java

```diff
@@ -133,49 +144,76 @@ else if (topType.equalsIgnoreCase("GTOP")) {
                         : NumberRange.inRangeToString(topValue, valueFormatting));
                 topFormat = topFormat.replace("{VALUE}", topValue.toString());
 
-                return formatGuildRank(config, tablistConfig, text, "{GTOP-" + comparatorType + "-" + index + "}", targetUser, guild, topFormat);
+                return this.formatGuildRank(text, "{GTOP-" + comparatorType + "-" + index + "}", targetUser, guild, topFormat);
             }
         }
 
         return text;
     }
 
     /**
-     * Parse legacy top placeholders (PTOP/GTOP-x) in text
+     * Format top position placeholders (POSITION/G-POSITION-type) in text.
      *
-     * @param targetUser user for which text will be parsed
-     * @param text       text to parse
-     * @return parsed text
+     * @param text       text to format
+     * @param targetUser user for which text will be formatted
+     * @return formatted text
      */
-    @Deprecated
-    public static String parseRank(User targetUser, String text) {
-        return parseRank(
-                FunnyGuilds.getInstance().getPluginConfiguration(),
-                FunnyGuilds.getInstance().getTablistConfiguration(),
-                FunnyGuilds.getInstance().getMessageConfiguration(),
-                FunnyGuilds.getInstance().getUserRankManager(),
-                FunnyGuilds.getInstance().getGuildRankManager(),
-                targetUser,
-                text
-        );
+    public String formatTopPosition(
+            String text,
+            @Nullable User targetUser
+    ) {
```

Comment:
Dlaczego to jest tak sformatowane

Reply:
wcześniej jak ten konstruktor był długi to było czytelniej, teraz można w sumie to zrobic w 1 lini